utils/sam/run.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+set -e
+set -u
+
+echo_usage() {
+	echo "usage: Deploy Lambda layer/application by SAM"
+	echo " -r <aws region>"
+	echo " -t <cloudformation template>"
+	echo " -b <sam build>"
+	echo " -d <deploy>"
+	echo " -n <specify layer name>"
+	echo " -l <show layer arn>"
+	echo " -s <stack name>"
+}
+
+is_sample() {
+	if [[ $(pwd) == *"sample"* ]]; then
+		echo 1
+	else
+		echo 0
+	fi
+}
+
+main() {
+	echo "running..."
+	saved_args="$@"
+	template='template.yml'
+	build=false
+	deploy=false
+	layer=false
+
+	region=${AWS_REGION-$(aws configure get region)}
+	stack=${KEY-"otel"}
```

Comment:
`KEY` seems like too generic of a name. Maybe `STACK`?

Reply:
Or OTEL_LAMBDA_STACK (OTEL_LAMBDA_KEY would be ok too)

Reply:
yeah, update to OTEL_LAMBDA_STACK